taiga/client.py

```diff
@@ -180,31 +186,12 @@ def auth_app(self, app_id, app_secret, auth_code, state=""):
             raise exceptions.TaigaRestException(full_url, 400, "NETWORK ERROR", "POST")
         if response.status_code != 200:
             raise exceptions.TaigaRestException(full_url, response.status_code, response.text, "POST")
-        cyphered_token = response.json().get("cyphered_token", "")
-        if cyphered_token:
-            from jwkest.jwe import JWE
-            from jwkest.jwk import SYMKey
-
-            sym_key = SYMKey(key=app_secret, alg="A128KW")
-            data, success = JWE().decrypt(cyphered_token, keys=[sym_key]), True
-            if isinstance(data, tuple):
-                data, success = data
-            try:
-                self.token = json.loads(data.decode("utf-8")).get("token", None)
-            except ValueError:  # pragma: no cover
-                self.token = None
-            if not success:
-                self.token = None
-        else:
-            self.token = None
-
-        if self.token is None:
+        token = response.json().get("token", None)
+
+        if token is None:
             raise exceptions.TaigaRestException(full_url, 400, "INVALID TOKEN", "POST")
 
-        self.raw_request = RequestMaker(
-            "/api/v1", self.host, self.token, "Application", self.tls_verify, proxies=self.proxies
-        )
-        self._init_resources()
```

Comment:
@heck-gd Why dropping this? Shouldn't `auth_app` not return anything, and just set `self.raw_request` and `self._init_resources()` as before, but with the new token?